it/google-cloud-platform/src/main/java/org/apache/beam/it/jdbc/StaticJDBCResource.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.it.jdbc;
+
+/** Parent class for Static JDBC Resources. */
+public abstract class StaticJDBCResource {
+  private final String hostname;
+  private final String username;
+  private final String password;
+  private final int port;
+
+  private final String database;
+
+  enum SourceType {
+    ORACLE,
+    MYSQL,
+    POSTGRESQL,
```

Comment:
[Super-nit, can be a todo]
In case the `it` folder is not covered under UT, could we exempt it for the Codecov tests?

Reply:
Added a todo comment